cuda/private/toolchain_configs/nvcc.bzl

```diff
@@ -387,6 +387,34 @@ def _impl(ctx):
         ],
     )
 
+    nvcc_relaxed_constexpr_feature = feature(
+        name = "nvcc_relaxed_constexpr",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = [
+                    ACTION_NAMES.cuda_compile,
+                    ACTION_NAMES.device_link,
+                ],
+                flag_groups = [flag_group(flags = ["--expt-relaxed-constexpr"])],
+            ),
+        ],
+    )
+
+    nvcc_extended_lambda_feature = feature(
+        name = "nvcc_extended_lambda",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = [
+                    ACTION_NAMES.cuda_compile,
+                    ACTION_NAMES.device_link,
+                ],
+                flag_groups = [flag_group(flags = ["--extended-lambda"])],
```

Comment:
Hmm, some old version of nvcc only supports [--expt-extended-lambda](https://docs.nvidia.com/cuda/archive/10.1/cuda-compiler-driver-nvcc/#options-for-altering-compiler-linker-behavior-expt-extended-lambda)

so you might use `["--extended-lambda" if nvcc_version_ge(ctx, some, version) else "--expt-extended-lambda"]`

Reply:
`nvcc_version_ge(ctx, 10, 2)`

https://docs.nvidia.com/cuda/archive/10.2/cuda-compiler-driver-nvcc/#options-for-altering-compiler-linker-behavior-expt-extended-lambda